packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselDefault.stories.tsx

```diff
@@ -27,23 +28,27 @@ const TestDiv = (text: string, bgColor: string) => {
 };
 
 export const Default = (props: Partial<CarouselProps>) => (
-  <div style={{ overflow: 'hidden' }}>
-    <Carousel circular={true} peeking={true} defaultValue={'test-1'} {...props}>
+  <Carousel circular={true} peeking={true} defaultValue={'test-1'} {...props}>
+    <CarouselSlider cardWidth={'50%'}>
```

Comment:
slick - I like that this is optional.

Curious - when does DOM get rendered? Thinking ahead to Wizard scenario where future steps change based on users settings AND are heavy on DOM.

Obviously this is a-okay for OOBE type stuff, but just thinking of edge cases.

Reply:
Good call, yeah if we have things that aren't yet rendered in DOM but need to be represented in nav we would likely have to take an approach more similar to Virtualizer, where the object has knowledge of steps etc. ahead of time.

The other option is to follow a similar path to this, and render all steps in the DOM but have any inactive steps be set to inert/hidden.